easybuild/easyblocks/t/trinity.py

```diff
@@ -72,6 +73,8 @@ def butterfly(self):
 
         self.log.info("Begin Butterfly")
 
+        setvar("-Dfile.encoding", "UTF8")
```

Comment:
This looks wrong to me...

`setvar` is for defining environment variables, you are defining `$-Dfile.encoding` to `UTF8` here, which is not what you want to do I'm guessing?

Reply:
oops..